File: src/reverse_string.py
# url: to the problem
# https://leetcode.com/problems/reverse-string/
# written by: hoangddt

class Solution:
    def reverseString(self, s):
        """
        :type s: str
        :rtype: str
        """
        res = ''
        for index in range(len(s)):
            res += s[len(s) - index -1]
        return res

File: src/reverse_string.js
// https://leetcode.com/problems/reverse-string/
// # written by: thanhvan
 
var reverseString = function(str) {
    var arrayofchars= str.split("");
    var StrtoArray= arrayofchars.reverse();
    var newstr= StrtoArray.join("");
    return newstr;
}
reverseString("hello");